### ISSUE TITLE
Bug: Link Account - Clicking on "Confirm" when configuring the Homebridge Config UI X instance does nothing when using HOOBS

### ISSUE THREAD

**Issue by mowens** [d0n4v4nb3ck3r/homebridge-honeywell-home#553]
### Describe The Bug

When attempting to link account the "Confirm" button does nothing. Possible HOOBS specific?

### To Reproduce

Follow all setup steps using HOOBS

### Expected behavior

Clicking confirm would continue to the next step in the setup process

### Relevant log output

```shell
Dashboard
Accessories
Log
Profiles
Bridges
Plugins
9/11/2021, 3:59:17 PMHoneywell Home BridgeInitializing HAP-NodeJS v0.9.4...9/11/2021, 3:59:44 PMHoneywell Home BridgeWarningNo plugins installed.9/11/2021, 3:59:44 PMHoneywell Home BridgePreparing Advertiser for 'Honeywell Home Bridge 551E' using bonjour-hap backend!9/11/2021, 3:59:44 PMHoneywell Home BridgeStarting to advertise 'Honeywell Home Bridge 551E' using bonjour-hap backend!9/11/2021, 3:59:44 PMHoneywell Home BridgeBridge is running on port 51866.9/11/2021, 4:00:50 PMHoneywell Home BridgeConfiguration change9/11/2021, 4:00:58 PMHoneywell Home BridgeInitializing HAP-NodeJS v0.9.4...9/11/2021, 4:01:29 PMHoneywell Home BridgeLoaded plugin 'homebridge-honeywell-home'9/11/2021, 4:01:29 PMHoneywell Home BridgeRegistering platform 'homebridge-honeywell-home.HoneywellHome'9/11/2021, 4:01:29 PMHoneywell Home BridgeLoading 1 platforms...9/11/2021, 4:01:29 PMHoneywell Home BridgeHoneywellHomeWarningUsing Default Refresh Rate.9/11/2021, 4:01:29 PMHoneywell Home BridgeHoneywellHomeWarningUsing Default Push Rate.9/11/2021, 4:01:29 PMHoneywell Home BridgeHoneywellHomeError"Missing Credentials"9/11/2021, 4:01:29 PMHoneywell Home BridgePreparing Advertiser for 'Honeywell Home Bridge 551E' using bonjour-hap backend!9/11/2021, 4:01:29 PMHoneywell Home BridgeStarting to advertise 'Honeywell Home Bridge 551E' using bonjour-hap backend!9/11/2021, 4:01:29 PMHoneywell Home BridgeBridge is running on port 51866.9/11/2021, 4:23:47 PMHoneywell Home BridgeConfiguration change9/11/2021, 4:23:55 PMHoneywell Home BridgeInitializing HAP-NodeJS v0.9.4...9/11/2021, 4:23:57 PMHoneywell Home BridgeLoaded plugin 'homebridge-honeywell-home'9/11/2021, 4:23:57 PMHoneywell Home BridgeRegistering platform 'homebridge-honeywell-home.HoneywellHome'9/11/2021, 4:23:57 PMHoneywell Home BridgeLoading 1 platforms...9/11/2021, 4:23:57 PMHoneywell Home BridgeHoneywellHomeWarningUsing Default Refresh Rate.9/11/2021, 4:23:57 PMHoneywell Home BridgeHoneywellHomeWarningUsing Default Push Rate.9/11/2021, 4:23:57 PMHoneywell Home BridgeHoneywellHomeError"Missing refreshToken"9/11/2021, 4:23:57 PMHoneywell Home BridgePreparing Advertiser for 'Honeywell Home Bridge 551E' using bonjour-hap backend!9/11/2021, 4:23:57 PMHoneywell Home BridgeStarting to advertise 'Honeywell Home Bridge 551E' using bonjour-hap backend!9/11/2021, 4:23:57 PMHoneywell Home BridgeBridge is running on port 51866.9/11/2021, 5:24:18 PMHoneywell Home BridgeInitializing HAP-NodeJS v0.9.4...9/11/2021, 5:24:20 PMHoneywell Home BridgeLoaded plugin 'homebridge-honeywell-home'9/11/2021, 5:24:20 PMHoneywell Home BridgeRegistering platform 'homebridge-honeywell-home.HoneywellHome'9/11/2021, 5:24:20 PMHoneywell Home BridgeLoading 1 platforms...9/11/2021, 5:24:20 PMHoneywell Home BridgeHoneywellHomeWarningUsing Default Refresh Rate.9/11/2021, 5:24:20 PMHoneywell Home BridgeHoneywellHomeWarningUsing Default Push Rate.9/11/2021, 5:24:20 PMHoneywell Home BridgeHoneywellHomeError"Missing refreshToken"9/11/2021, 5:24:20 PMHoneywell Home BridgePreparing Advertiser for 'Honeywell Home Bridge 551E' using bonjour-hap backend!9/11/2021, 5:24:20 PMHoneywell Home BridgeStarting to advertise 'Honeywell Home Bridge 551E' using bonjour-hap backend!9/11/2021, 5:24:20 PMHoneywell Home BridgeBridge is running on port 51866.
```

### Config for homebridge-honeywell-home

```shell
{
    "platform": "HoneywellHome",
    "name": "HoneywellHome",
    "credentials": {
        "consumerKey": "<removed>",
        "consumerSecret": "<removed>"
    }
}
```

### Screenshots

<img width="971" alt="Screen Shot 2021-09-11 at 5 47 40 PM" src="https://user-images.githubusercontent.com/4575216/132967108-29493103-8027-446b-9c42-ff61b57f3dd6.png">

### Device & Model

N/A

### Node.js Version

14.17.6

### NPM Version

6.14.15

### Homebridge Version

1.3.4

### Homebridge Honeywell Home Plugin Version

10.0.1

### Homebridge Config UI X Plugin Version

Unsure

### Operating System

Raspbaian (via HOOBS)

**Comment by donavanbecker** [comment]
You will need to[ move to homebridge](https://github.com/homebridge/homebridge-raspbian-image/wiki/How-To-Migrate-From-HOOBS-To-Homebridge).

I only develop this plugin for homebridge.

**Comment by mowens** [comment]
Are the refreshToken and accessToken that are created when linking an account static or do they need to change over time? Since HOOBS runs Homebridge under the hood I was able to successful setup Homebridge locally to perform the setup process such that I was able to get the JSON configuration, then copy that over to HOOBS to get everything to show up. The thermostats reports the state correctly, however, trying to change the state from OFF, HEAT, COOL does result in an error

```
9/12/2021, 12:55:27 AMHoneywell Home BridgeConfiguration change
9/12/2021, 12:55:42 AMHoneywell Home BridgeHoneywellHomeSending request for Downstairs Thermostat Thermostat to Honeywell API. mode: Cool coolSetpoint: 78 heatSetpoint: 70 thermostatSetpointStatus: PermanentHold
9/12/2021, 12:55:44 AMHoneywell Home BridgeERROR[Honeywell Home Bridge 551E@Downstairs Thermostat Thermostat@@Target Heating Cooling State] characteristic value 3 is not contained in valid values array
```

I can understand that this is only technically supported on homebridge as that is the platform you are developing this for but is this a known issue on the supported homebridge side as well? Based on this thread it seems like 3 AUTO isn't a supported value by the spec:

https://github.com/homebridge/HAP-NodeJS/issues/815
